internal/target.go

```diff
@@ -43,14 +45,19 @@ func (t Target) Scan(ctx context.Context) (sbom.SBOM, error) {
 		return sbom.SBOM{}, fmt.Errorf("failed to get source from %q: %w", t.Path, err)
 	}
 
+	sr := pkgcataloging.NewSelectionRequest().
+		WithDefaults(pkgcataloging.ImageTag).
+		WithAdditions("sbom-cataloger")
+
+	if v, ok := os.LookupEnv("BUILDKIT_SCAN_SELECT_CATALOGERS"); ok {
```

Comment:
I think we should make the env var consistent with what we currently have: https://docs.docker.com/build/metadata/attestations/sbom/#arguments

What do you think about:

```suggestion
	if v, ok := os.LookupEnv("BUILDKIT_SBOM_SCAN_CATALOGERS"); ok {
```

cc @dvdksn as this needs some docs changes.

Reply:
If we want that handled in buildkit I guess it needs a follow-up on the frontend: https://github.com/moby/buildkit/blob/f60ac4a15d4e71974eb064ec2ab06e02487a98b8/frontend/dockerfile/dockerfile2llb/convert.go#L56-L57